src/torcontrol.cpp

```diff
@@ -662,26 +662,26 @@ void TorController::reconnect_cb(evutil_socket_t fd, short what, void *arg)
 }
 
 /****** Thread ********/
-struct event_base *base;
```

Comment:
Oh no, so we're at renaming random variables now without even understanding why just to shut up warnings.